bwpy/mea_viewer.py

```diff
@@ -0,0 +1,77 @@
+from . import signal
+import abc
+import plotly.graph_objects as go
```

Comment:
Can you move this import to inside the function that uses the plotly objects? Wrap in `try/catch ImportError` and raise an error that the user should install plotly to use the viewer. Otherwise we have to add plotly as a hard dependency of the package, and I don't want that :)